SimpleRss/SimpleRss/ViewControllers/FeedViewController.swift

```diff
@@ -90,7 +93,9 @@ extension FeedViewController: UITableViewDelegate {
         
         guard let cell = tableView.cellForRow(at: indexPath) as? FeedViewCell else { return }
         
-        cell.descriptionLabel.numberOfLines = cell.descriptionLabel.numberOfLines == 0 ? 1 : 0;
+        feedList[indexPath.row].isExpanded = !feedList[indexPath.row].isExpanded
```

Comment:
Add a function 'toggle' for FeedViewModel to change its 'isExpanded' flag.
Make setter of isExpanded variable private.